release-engineering/role-handbooks/branch-manager.md

```diff
@@ -187,25 +185,16 @@ This can be done in one of two ways:
 [k-announce-request]: https://groups.google.com/forum/#!contactowner/kubernetes-announce
 [release tools]: https://github.com/kubernetes/release
 
-### Safety Check
-
-The following command can help verify that your clone of the [release tools] are up-to-date, that your setup is fully configured, and your `gcloud` identity is authorized for the release builds:
-
-```shell
-./gcbmgr
-```
-
-This command should run successfully and even show some green colored "OK" words
-
```

Comment:
Do we want to add such a pre-check to krel as well?

Reply:
I think the safety check was in place to validate if you have the tools installed, like `jq` and the access to gcp

Now you need to have the git configured and access to gcp right? not other external dependencies. is so maybe we can add the pre-check to validate that, not sure if needed

Reply:
looks like i am wrong
```
INFO Checking for required packages
WARN Unable to exec: "dpkg": executable file not found in $PATH
WARN Unable to exec: "rpm": executable file not found in $PATH
WARN Unable to exec: "pacman": executable file not found in $PATH
FATA unable to verify if packages are available: unable to find working package manager

```

Reply:
Yeah okay I guess we're fine with the current checks. On which distribution are you on?